test/admin/e2e/general/e2e.spec.ts

```diff
@@ -1024,6 +1024,38 @@ describe('General', () => {
     })
   })
 
+  test('should not open leave-without-saving modal if opening a new tab', async () => {
+    const { id } = await createPost()
+    await page.goto(postsUrl.edit(id))
+    const title = 'title'
+    await page.locator('#field-title').fill(title)
+    await saveDocHotkeyAndAssert(page)
+    await expect(page.locator('#field-title')).toHaveValue(title)
+
+    const newTitle = 'new title'
+    await page.locator('#field-title').fill(newTitle)
+
+    // Open link in a new tab by holding down the Meta key
+    const [newPage] = await Promise.all([
+      page.context().waitForEvent('page'),
+      page
+        .locator('header.app-header a[href="/admin/collections/posts"]')
+        .click({ modifiers: ['Meta'] }),
+    ])
+
+    await newPage.waitForLoadState('domcontentloaded')
+
+    // Locate the modal container and ensure it is not visible
+    const modalContainer = page.locator('.payload__modal-container')
+    await expect(modalContainer).toBeHidden()
+
+    // Ensure the new page is the correct URL
+    expect(newPage.url()).toContain('/admin/collections/posts')
```

Comment:
Same as above. Can standardize the url matching behind the `postsUrl.list` utility.